python-backend/app/api/now_submissions/resources/application_resource.py

```diff
@@ -18,3 +27,42 @@ def get(self, application_guid):
             raise NotFound('Application not found')
 
         return application
+
+
+class ApplicationListResource(Resource, UserMixin, ErrorMixin):
```

Comment:
I'd suggest that this go in its own resource file. Namely because POST and GET all (at the `/applications` endpoint) will be handled in one file/resource; and PUT, GET one, and DELETE (the `/applications/{application_guid}` endpoint) will be handled in the other.

It's not necessarily an objectively right way to do it, but I found it worked well while building out Variances, as per feedback I got from Jason S.